source/includes/api/_graph.md

```diff
@@ -1,583 +1,865 @@
+-H 'Content-Type: application/json' \
+-H 'Authorization: Bearer <your-token>' \
+-d '{
+  "application": "rw",
+  "tags": ["urban"]
+}'
+```
+
+> Deleting the association between the vocabulary "knowledge_graph" and a dataset:
+
+```shell
+curl -X DELETE https://api.resourcewatch.org/v1/dataset/:id/vocabulary/knowledge_graph
+```
+
+**The management of connections between resources and concepts is handled by vocabulary endpoints, using the vocabulary `"knowledge_graph"`.** Using this vocabulary, you can add tags to your dataset. These tags will be added as concepts, and one (or more, accordingly) graph edges will be created, establishing a connection between your resource and the provided tags.
+
+This section provides some examples of how you can use vocabulary endpoints to manage your resources' concepts. Keep in mind that you can always refer to the [vocabulary endpoint documentation](#vocabulary-and-tags) for more details on how to use these endpoints.
+
+On the examples on the side, you'll be able to understand that you can map the tags associated with a given resource with the tags associated with the `"knowledge_graph"` vocabulary for the same resource. You can use vocabulary's endpoints to create (if it doesn't exist yet), edit, or delete the tags associated with that resource. Those changes will be reflected in the concepts that are associated with that same resource.
+
+Lastly, keep in mind that, despite the examples on the side refer to datasets, you can use vocabulary's endpoints to update the tags associated with the `"knowledge_graph"` vocabulary for all supported resource types: datasets, layers, widgets, and metadata.
+
+## Favorite relationships between graph nodes and users
+
+As in the case of managing relationships between graph nodes and concepts, **the management of favorite relationships between resources and users is handled by vocabulary endpoints.** Please refer to the [favorite endpoint documentation](#favorites) for more details on how to use these endpoints.
+
+## Graph concept reference
+
+> Example concept entity structure:
+
+```json
+{
+  "id": "society",
+  "label": "Society",
+  "synonyms": ["People"],
+  "labels": ["CONCEPT", "TOPIC"],
+  "numberOfDatasetsTagged": 1,
+  "datasets": ["4458eb12-8572-45d1-bf07-d5a3ee097021"]
+}
+```
+
+This section describes the attributes that are present on a graph concept entity:
+
+Field name               | Type           | Description                                                                  
+------------------------ | -------------- | ---------------------------------------------------------------------------- 
+`id`                     | String         | The concept unique identifier.
+`label`                  | String         | A readable version of the concept identifier.
+`synonyms`               | String | Array | The list of synonyms for this concept (or an empty string, if no synonyms exist).
```

Comment:
Table syntax error here - the pipe in `String | Array` puts array in the third column and cuts off the description